src/components/breadcrumb.jsx

```diff
@@ -0,0 +1,19 @@
+const Breadcrumb = ({ links }) => {
+  return (
+    links && (
+      <nav role="navigation" aria-labelledby="system-breadcrumb">
+        <h2 id="system-breadcrumb" className="visually-hidden">
```

Comment:
It might actually be better to add `aria-label="Breadcrumbs"` to the `nav` element and get rid of the `h2` entirely. I can see some complications with using an `h2` heading hidden offscreen because the headings may become out of order if this is rendered above an `h1`. Normally to get around that, we'd use css to reposition so this `h2` comes after, but that's complicated for the drag and drop use case of the users.